parm/config/config.efcsfsoi

```diff
@@ -0,0 +1,96 @@
+#!/bin/ksh -x
+
+########## config.efcsfsoi ##########
+# Ensemble forecast specific, dependency: config.fcst
+
+echo "BEGIN: config.efcsfsoi"
+
+# Get regular efcs configs
+. $EXPDIR/config.efcs
+
+# Get task specific resources
+. $EXPDIR/config.resources efcsfsoi
+
+export npe_fv3=$npe_efcsfsoi
+
+if [ $QUILTING = ".true." ]; then
+    export npe_fv3=$(echo " $npe_fv3 + $WRITE_GROUP * $WRTTASK_PER_GROUP" | bc)
+    export npe_efcsfsoi=$npe_fv3
+fi
+
+### Number of enkf members per fcst job
```

Comment:
Why is there this huge block of commented out code?

Reply:
removed